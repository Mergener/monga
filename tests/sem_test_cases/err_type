function foo(): int {
	return 50.3;
}