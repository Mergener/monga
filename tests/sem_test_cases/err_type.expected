[Error] At line 2, column 14: Cannot return value of type 'float' in function declared with return type 'int'.
Semantic errors were caught during analysis.
